plasmapy/classes/sources/analyticalfields.py

```diff
@@ -0,0 +1,47 @@
+"""
+Defines the AnalyticalFields class used by PlasmaPy to represent electric and magnetic fields varying
+as known functions of position.
+"""
+import astropy.units as u
+
+from plasmapy.classes import GenericPlasma
+import numpy as np
+
+__all__ = [
+    "AnalyticalFields"
+]
+
+
+class AnalyticalFields(GenericPlasma):
+    """
+    Work-in-progress class for passing analytical functions as fields 
+
+    This is primarily helpful for `plasmapy.simulation.ParticleTracker`.
+    """
+
+    def __init__(self, magnetic_field, electric_field):
+        """
+        Initialize plasma paramters.
+        The most basic description is composition (ion), temperature,
+        density, and ionization.
+        """
```

Comment:
The plasma parameters in the docstring don't match the arguments to `__init__` here.

Reply:
Oh yeah, it's a bad habit of mine that I don't update docs as I go. We should really find a way to incorporate pydocstyle into CI, or maybe I should finally learn pre-commit...